templates/consent.html

```diff
@@ -0,0 +1,32 @@
+<amp-geo layout="nodisplay">
```

Comment:
Can you only add this if amp-consent and amp-geo are not already present on the page:

```
{{^includesconsent}}
 {{^includesgeo}}
<amp-geo layout="nodisplay">
  <script type="application/json">
    {
      "isocountrygroups": {
        "eu": ["at", "be", "bg", "ch", "cy", "cz", "dk", "de", "ee", "el", "es", "fi", "fr", "hr", "ie", "is", "it", "li", "lv", "lt", "lu", "hu", "mt", "nl", "no", "pl", "pt", "ro", "si", "sk", "se", "uk"]
      }
    }
  </script>
</amp-geo>

<amp-consent id="consent" layout="nodisplay">
  <script type="application/json">
    {
      "consents": {
        "eu": {
          "promptifunknownforgeogroup": "eu",
          "promptui": "consent-popup"
        }
      }
    }
  </script>

  <div id="consent-popup" class="consent-popup">
    <div>
      <a href="#" on="tap:consent.dismiss" class="close-button" role="button" tabindex="0">✕</a>
      <p>
        we use cookies to understand how you use our site and to improve your experience. by continuing to use our site, you accept our <a href="https://policies.google.com/technologies/cookies">use of cookies</a> and <a href="https://policies.google.com/privacy">privacy policy</a>.
      </p>
      <a class="ampstart-btn ampstart-btn-secondary caps text-decoration-none inline-block" on="tap:consent.accept" class="button text-label" role="button" tabindex="1">ok</a>
    </div>
  </div>
</amp-consent>
 {{/includesgeo}}
 {{/includesconsent}}
```

Reply:
done!